rpc-services/qubesbuilder.ExportDisk

```diff
@@ -13,7 +13,7 @@ read -r -n 1024 path
 [[ "${#path}" -lt 1024 ]]
 
 if [ -z "$key" ]; then
-    key=$( head -c 20 /dev/urandom | xxd -p )
+    key=$( head -c 256 /dev/urandom | xxd -p )
```

Comment:
If going this extreme, you need also `-c0`, otherwise `xxd` will break line every 30 bytes.

Please also carefully verify if the key is not used in qrexec call argument (which on R4.0 is limited together with the service name to 64 characters). I think it is not, but double checking wont hurt.

Reply:
It isn’t used anywhere.  More important is replacing key with a hash in `qubesbuilder.CopyTemplateBack`.